h2o-automl/src/test/java/ai/h2o/automl/ModelSelectionStrategiesTest.java

```diff
@@ -75,7 +76,7 @@ public void prepareModels() {
         leaderboardSupplier = () -> {
             String name = "selection_lb";
             EventLog el = EventLog.getOrMake(Key.make(name)); //toDelete.add(el);
-            Leaderboard lb = Leaderboard.getOrMake(name, el, fr, "logloss"); //toDelete.add(lb);
+            Leaderboard lb = Leaderboard.getOrMake(name, el.asLogger(EventLogEntry.Stage.Workflow),  fr, "logloss"); //toDelete.add(lb);
```

Comment:
for consistency, even in tests, let's use `ModelTraining` stage everywhere (also valid for other tests below)